devtools/pip-tools/pr_tool/main.py

```diff
@@ -184,7 +184,7 @@ def _check_headline(cls, text):
         issues = []
         # we encourage to use no more than 50 chars, but still accept up to 60
         if len(text) > 60:
-            issues.append("headline too long")
+            issues.append("headline too long ({} > 60)".format(len(text)))
```

Comment:
The 60 should get named, so we cannot accidentally only change it in one place
```
max_headline_length = 60
if len(text) > max_headline_length
    issues.append("headline too long ({} > {})".format(len(text), max_headline_length))
	
```

Reply:
done.